scripts/helpers.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+load helpers
+
+teardown() {
+	unset DEV_TAGS
+}
+
+@test "assert valid dev_tags set" {
+	export DEV_TAGS='
+		hashicorppreview/repo1:1.2.3
+		public.ecr.aws/dadgarcorp/repo1:1.2.3
+	'
+
+	run dev_tags_validation "$DEV_TAGS"
+	[ $status -eq 0 ]
+}
+
+@test "assert invalid dev_tags set with no host" {
+	export DEV_TAGS='
+		dadgarcorp/repo1:1.2.3
+		public.ecr.aws/dadgarcorp/repo1:1.2.3
+	'
+
+	run dev_tags_validation "$DEV_TAGS"
+	[ $status -eq 1 ]
+	# [ "$output" = "Only valid DockerHub organization is 'hashicorppreview' (Got: dadgarcorp/repo1:1.2.3)" ]
+}
+
+@test "assert invalid dev_tags set with host docker.io" {
+	export DEV_TAGS='
+		docker.io/dadgarcorp/repo1:1.2.3
+		public.ecr.aws/dadgarcorp/repo1:1.2.3
+	'
+
+	run dev_tags_validation "$DEV_TAGS"
+	[ $status -eq 1 ]
+	# [ "$output" = "Only valid DockerHub organization is 'hashicorppreview' (Got: docker.io/dadgarcorp/repo1:1.2.3)" ]
+}
```

Comment:
Thanks for adding these tests! It would be great to assert on partial output as well, to validate that we're hitting the error we expect. I can see that here, commented out. Rather than `=` which gets tripped up by whitespace etc, maybe we could try `~=` with wildcards to check that the error message is _present_ in the output?

We should ideally also put the error message on stderr as noted above, I believe BATS puts all output together in `$output` though so you should still be able to check it using that var.

Reply:
This was commented out during testing and I forgot to add it back!